README.md

```diff
@@ -1,4 +1,161 @@
+[Personal Package Archives](https://launchpad.net/ubuntu/+ppas) (PPA) files.
+They are built and hosted by [Launchpad](https://launchpad.net/) to the
+specifications (recipes) provided here.
+
+| Version      | SDK Binary                                                                                           | Source                                                                                          |
+|--------------|------------------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------|
+| Stable Build | [Launchpad PPA packages](https://code.launchpad.net/~kivy-team/+archive/ubuntu/kivy/+packages)       | [Launchpad recipes](https://code.launchpad.net/~kivy-team/+archive/ubuntu/kivy/+packages)       |
+| Daily Build  | [Launchpad PPA packages](https://code.launchpad.net/~kivy-team/+archive/ubuntu/kivy-daily/+packages) | [Launchpad recipes](https://code.launchpad.net/~kivy-team/+archive/ubuntu/kivy-daily/+packages) |
+
+[Kivy SDK for Linux README](linux/debian/README.md)
+
+## macOS
+
+Kivy SDKs for macOS are built as Disk Image (DMG) files.
+
+These can be built on the developer's machine. 
+
+> [!NOTE]
+> A Kivy.app build on one macOS version will typically not work on earlier
+> macOS versions. You need to build Kivy SDKs for macOS on the oldest machine 
+> you wish to support.
+
+[Buildozer](https://buildozer.readthedocs.io) builds and uses Kivy SDKs to 
+package macOS apps.
+
+[![macOS Tests](https://github.com/kivy/kivy-sdk-packager/actions/workflows/test_macos.yaml/badge.svg)](https://github.com/kivy/kivy-sdk-packager/actions/workflows/test_macos.yaml)
+
+[Kivy SDK for macOS README](osx/README.md)
+
+## Windows
+
+Kivy SDK for Windows are built as wheels that can be installed via 
+[pip](https://pypi.org/project/pip/). They are uploaded and hosted on the 
+[Python Package Index](https://pypi.org/) (PyPI).
+
+Four variants are released, based on four different 
+[OpenGL ES](https://en.wikipedia.org/wiki/OpenGL_ES) implementations:
+
+| Version                                                                 | PyPI Name                                                            |
+|-------------------------------------------------------------------------|----------------------------------------------------------------------|
+| [Angle](https://chromium.googlesource.com/angle/angle/+/main/README.md) | [kivy-deps.angle](https://pypi.org/project/kivy-deps.angle/)         |
+| [Glew](https://glew.sourceforge.net/)                                   | [kivy-deps.glew](https://pypi.org/project/kivy-deps.glew/)           |
+| [Gstreamer](https://gstreamer.freedesktop.org/)                         | [kivy-deps.gstreamer](https://pypi.org/project/kivy-deps.gstreamer/) |
+| [SDL2](https://www.libsdl.org/)                                         | [kivy-deps.sdl2](https://pypi.org/project/kivy-deps.sdl2/)           |
```

Comment:
This is not true 😅

See:
- https://kivy.org/doc/stable/api-kivy.graphics.cgl.html
- https://kivy.org/doc/stable/gettingstarted/installation.html#installing-kivy-s-dependencies